modules/raft/src/main/java/org/apache/ignite/raft/jraft/core/FSMCallerImpl.java

```diff
@@ -665,6 +676,27 @@ private void doSnapshotLoad(final LoadSnapshotClosure done) {
             setError(e);
             return;
         }
+
+        // Tests use such strange metas, so we have to protect... in production, these are never null.
```

Comment:
What tests? Are they ours or from Jraft?

Reply:
JRaft tests. I've clarified the comment.